### PR TITLE
change App id article

### PR DESCRIPTION
**Pull request by tsonevn**
change App id article

### CODE REVIEW

**Review by NickIliev — APPROVED** [review]

File: docs/tooling/changing-appid.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: Change App ID
+description: Learn how change the application identifire(App ID) in the NativeScript application for both platforms(iOS/Android).
```
Comment (KristianDD) [inline docs/tooling/changing-appid.md]: identifier

File: docs/tooling/changing-appid.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+position: 60
+slug: changing-appid
+---
+
+# What is App identifier
+
+The App ID is a unique string, which contain two or more strings, separated by a dot. It is used to identify your app on the device and in the store(iTunes Store / Google Play Store), where it is published. The application identifier refers to the iTunes ID and Google Play ID based on the mobile OS the app is built for. Each string must start with a letter and should contain only letters and numbers. The app identifier must not begin with an uppercase letter. In NativeScript we will use the project name, which is set, while executing the create command (`tns create <project_name>`), for the initially App ID.
```
Comment (KristianDD) [inline docs/tooling/changing-appid.md]: I am not sure about the term "iTunes ID". Haven't found any information that this exists. There is App ID and Bundle ID. The App ID is a combination of the Team ID and Bundle ID. Our application identifier is actually representing the Bundle ID.([source](https://developer.apple.com/library/archive/documentation/General/Conceptual/DevPedia-CocoaCore/AppID.html))
Also didn't find any information on the term "Google Play ID". The variable is called "application ID" - more information [here](https://developer.android.com/studio/build/application-id).
"for the **initially** App ID" ->  "for the **initial** App ID" might also add example like ("org.nativescript.{appName}")

File: docs/tooling/changing-appid.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+---
+
+# What is App identifier
+
+The App ID is a unique string, which contain two or more strings, separated by a dot. It is used to identify your app on the device and in the store(iTunes Store / Google Play Store), where it is published. The application identifier refers to the iTunes ID and Google Play ID based on the mobile OS the app is built for. Each string must start with a letter and should contain only letters and numbers. The app identifier must not begin with an uppercase letter. In NativeScript we will use the project name, which is set, while executing the create command (`tns create <project_name>`), for the initially App ID.  
+
+
+## How to change the App ID
+
+From NativeScript(v.5.0) we can change the App ID using the project's `package.json` file while specifying a unique string for iOS and Android. Follow the below-attached steps to change the identifier:
```
Comment (KristianDD) [inline docs/tooling/changing-appid.md]: NativeScript(v.5.0) -> NativeScript(v5.0.0).

**Review by KristianDD — CHANGES_REQUESTED** [review]
Maybe you can add that if the user doesn't need different id-s for each platform, he can leave it as a string value.

**Review by KristianDD — APPROVED** [review]
I still think that we should add that the new structure is needed only if the user needs different app id for the platforms.
```
"nativescript": {
    "id": {
        "ios": "{{appID1}}" ,
        "android": "{{appID2}}"
    }
  }
```
or 
```
"nativescript": {
    "id": "{{appID}}"
}
```

File: docs/tooling/changing-appid.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
+---
+title: Change App ID
+description: Learn how change the application identifier(App ID) in the NativeScript application for both platforms(iOS/Android).
+position: 60
+slug: changing-appid
+---
+
+# What is App identifier
+
+The App ID is a unique string, which contain two or more strings, separated by a dot. It is used to identify your app on the device and in the store(iTunes Store / Google Play Store), where it is published. The application identifier refers to the Bundle ID for iOS and application ID - Android based on the mobile OS the app is built for. Each string must start with a letter and should contain only letters and numbers. The app identifier must not begin with an uppercase letter. In NativeScript we will use the project name, which is set, while executing the create command (`tns create <project_name>`), for the initial App ID(e.g. `org.nativescript.{appName}`).
```
Comment (etabakov) [inline docs/tooling/changing-appid.md]: I think the appId cannot start with number as well, at least for Android

File: docs/tooling/changing-appid.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+}
+```
+Example:
+```
+"nativescript": {
+    "id": "org.nativescript.myappid"
+}
+```
+
+2.  If you are using old project, open `<project_name>/App_Resources/Android/app.gradle` file and remove the `applicationId` key.
```
Comment (etabakov) [inline docs/tooling/changing-appid.md]: Let's state explicitly what "old" means. Before 5.0 I presume.

File: docs/tooling/changing-appid.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
+---
+title: Change App ID
+description: Learn how change the application identifier(App ID) in the NativeScript application for both platforms(iOS/Android).
+position: 60
+slug: changing-appid
+---
+
+# What is App identifier
+
+The App ID is a unique string, which contain two or more strings, separated by a dot. It is used to identify your app on the device and in the store(iTunes Store / Google Play Store), where it is published. The application identifier refers to the Bundle ID for iOS and application ID - Android based on the mobile OS the app is built for. Each string must start with a letter and should contain only letters and numbers. The app identifier must not begin with an uppercase letter or number. In NativeScript we will use the project name, which is set, while executing the create command (`tns create <project_name>`), for the initial App ID(e.g. `org.nativescript.{appName}`).
```
Comment (etabakov) [inline docs/tooling/changing-appid.md:10]:
```suggestion
The App ID is a unique string, which contain two or more strings, separated by a dot. It is used to identify your app on the device and in the store (iTunes Store / Google Play Store), where it is published. The application identifier refers to the Bundle ID for iOS and application ID - Android based on the mobile OS the app is built for. Each string must start with a letter and should contain only letters and numbers. The app identifier must not begin with an uppercase letter or number. In NativeScript we will use the project name, which is set, while executing the create command (`tns create <project_name>`), for the initial App ID (e.g. `org.nativescript.{appName}`).  
```

File: docs/tooling/changing-appid.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+---
+
+# What is App identifier
+
+The App ID is a unique string, which contain two or more strings, separated by a dot. It is used to identify your app on the device and in the store(iTunes Store / Google Play Store), where it is published. The application identifier refers to the Bundle ID for iOS and application ID - Android based on the mobile OS the app is built for. Each string must start with a letter and should contain only letters and numbers. The app identifier must not begin with an uppercase letter or number. In NativeScript we will use the project name, which is set, while executing the create command (`tns create <project_name>`), for the initial App ID(e.g. `org.nativescript.{appName}`).  
+
+
+## How to change the App ID
+
+From NativeScript(v5.0.0) we can change the App ID using the project's `package.json` file while specifying a unique string for iOS and Android. Follow the below-attached steps to change the identifier:
```
Comment (etabakov) [inline docs/tooling/changing-appid.md]:
```suggestion
From NativeScript (v5.0.0) we can change the App ID using the project's `package.json` file while specifying a unique string for iOS and Android. Follow the below-attached steps to change the identifier:
```

File: docs/tooling/changing-appid.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+}
+```
+Example:
+```
+"nativescript": {
+    "id": "org.nativescript.myappid"
+}
+```
+
+2.  If you are using project, created before NativeScript(v5.0.0), open `<project_name>/App_Resources/Android/app.gradle` file and remove the `applicationId` key.
```
Comment (etabakov) [inline docs/tooling/changing-appid.md]:
```suggestion
2.  If you are using project, created before NativeScript (v5.0.0), open `<project_name>/App_Resources/Android/app.gradle` file and remove the `applicationId` key.
```

**Review by etabakov — APPROVED** [review]
Looks great, just some whitespaces before opening brackets.

**Comment by tsonevn** [comment]
test

**Pull request merged**